pyrelational/datasets/classification.py

```diff
@@ -243,26 +247,27 @@ class FashionMNIST(Dataset):
             splits to compute
     """
 
-    def __init__(self, data_dir="/tmp/", n_splits=5):
+    def __init__(self, data_dir: str = "/tmp/", n_splits: int = 5):
```

Comment:
hmmm, possibly, does Path encompass strings? or would we need to convert strings everytime?

Reply:
Can we do `Union[str, pathlib.Path]`?

Reply:
hmm, apparently not. This is because all the operations within the code are defined for a string. Not a path

Reply:
e.g. summation is not defined between Path and str

Reply:
I see, we can keep as string for now, it will certainly work on UNIX based systems just fine, it will just cause some problems for Windows users when they use the default setting. I will make an issue for this for later.